docs/content/misc/faq.ngdoc

```diff
@@ -65,6 +58,44 @@ is also deprecated but we only remove the feature once it's removed from jQuery
 jqLite and jQuery. One such example is the `bind` method, deprecated in favor of `on` but unlikely to be removed
 from jqLite any time soon.
 
+#### What is the version compatibility between AngularJS main and optional modules?
+
+AngularJS code is separated into a main module ("angular"), and a few different optional modules
+("angular-animate", "angular-route" etc) that are dependant on the main module.
+When a new AngularJS version is released, all modules are updated to the new version.
+This means that the main module and the optional modules must always have the exact same version,
+down the patch number, otherwise your application might break.
+
+Therefore you must always explicitly lock down your dependencies, for example in the package.json,
+the following ensures that "angular" and "angular-animate" are always updated to the same version.
+
+```
+{
+  "angular": "~1.6.0",
+  "angular-animate": "~1.6.0"
+}
```

Comment:
Yes, to indicate that if you lock to a specific version, all versions need to be the same.